suse_ha-formula/suse_ha/resources.sls

```diff
@@ -36,3 +36,13 @@ ha_resources_directory:
 {%- else %}
 {%- do salt.log.debug('Skipping construction of custom resources') %}
 {%- endif %}
+
+{%- for constraint, config in constraints.items() %}
+{{ ha_constraint(
+      constraint, config.get('type'),
```

Comment:
Nitpick: you have one item per line in the following lines, but two here. I'd recommend to split this line.

Reply:
I was orienting at the formatting of the  `ha_resource()` call which keeps the name and type on one line, but can split here if you prefer.

Reply:
the `ha_resource()` call has 3, 4 and 1 argument per line, so it's at least visible that it's not one item per line.

I'd prefer to have the split.

Thinking about it - maybe it would be a good idea to also split the `ha_resource()` call to one parameter per line (as a separate commit or a separate PR). A quick test shows that it would be much easier to read:

```
{{ ha_resource(
    resource,
    config.get('class', 'ocf'),
    config.get('type', None),
    config.get('attributes', {}),
    config.get('operations', {}),
    config.get('meta_attributes', {}),
    config.get('provider', 'heartbeat'),
    config.get('clone', {})
  )
}}
```